docker.sh

```diff
@@ -87,9 +102,11 @@ getToken() {
 
     if [ "$REGISTRY" == "index.docker.io" ]; then
         if [ "$VERBOSE" == "true" ]; then
-            echo curl -s "https://auth.docker.io/token?service=registry.docker.io&scope=repository:$REPO:pull" > "`tty`"
+            echo "curl -s \"https://auth.docker.io/token?service=registry.docker.io&scope=repository:$REPO:pull\"" > "`tty`"
         fi
         curl -s "https://auth.docker.io/token?service=registry.docker.io&scope=repository:$REPO:pull" | jq -r '.token'
+    elif [ "$REGISTRY" == "ghcr.io" ]; then
+        echo $GITHUB_TOKEN | base64
```

Comment:
What If I am using directly the "secrets.GITHUB_TOKEN"?
According to this GitHub description: https://docs.github.com/en/actions/security-guides/automatic-token-authentication
the token is directly used without base64 encoding: `--header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \`

Maybe we can have a check here and try to detect a base64 encoded string. If it is already base64 encoded we might want to skip this step.

Reply:
I agree with you but iirc it didn't work without the base64 encoding. Not sure if it makes sense to try to implement a check without a concrete issue or use case - and thus not being able to test it either. Do you see any reasons why this should break in the future without the check?

Reply:
I created a `no-base64-github-token` branch and gave it a try. Without `| base64` authentication fails. See
<https://github.com/georg-jung/BlazorWoL/actions/runs/4041870109>. Thus, I don't see any advantage in changing this.

Reply:
hm, I see. Then I agree with you: adding the base64 check is not required

I tried it with the `secrets.GITHUB_TOKEN` but without success on accessing the private package of the repository.
Is the `secrets.GITHUB_TOKEN` working for you if you access the private package of your repository?

Reply:
Not sure if I get right what you're asking for. I tried to use the pipeline-specific automatically created GitHub token to access a private package that belongs to the repo the workflow belongs to. That didn't work.